apps/retrospective/models/retrospective.go

```diff
@@ -90,3 +92,26 @@ func RetroJoinTasks(db *gorm.DB) *gorm.DB {
 func RetroJoinUserTeams(db *gorm.DB) *gorm.DB {
 	return db.Joins("JOIN user_teams ON retrospectives.team_id = user_teams.team_id AND user_teams.deleted_at IS NULL")
 }
+
+// GetTaskTrackerConnectionFromRetro ...
+func GetTaskTrackerConnectionFromRetro(db *gorm.DB, retroID string) (tasktracker.Connection, error) {
+	var retro Retrospective
+	if err := db.Model(Retrospective{}).
+		Where("retrospectives.deleted_at IS NULL").
+		Where("id = ?", retroID).
+		Find(&retro).Error; err != nil {
+		utils.LogToSentry(err)
+		return nil, errors.New("couldn't find any retrospective with this ID")
```

Comment:
`couldn't find any retrospective with this ID` what ID?
change to `retrospective with ID <value> not found`